app/controllers/find/application_controller.rb

```diff
@@ -10,6 +12,16 @@ def render_feedback_component
       @render_feedback_component = true
     end
 
+    # For DfE::Analytics as Find doesn't have a current_user
+    def current_user
```

Comment:
You can leave this out as `dfe-analytics` will check whether the method exists before calling it:

https://github.com/DFE-Digital/dfe-analytics/blob/b916b441d1a5579465c52d0772d99dab8e072d73/lib/dfe/analytics/requests.rb#L21